db/sqldb.go

```diff
@@ -295,6 +291,19 @@ func (db SQLDatabase) ClearTables() error {
 	})
 }
 
+func (db SQLDatabase) getDomain(sqlQuery string, args ...interface{}) (models.Domain, error) {
```

Comment:
This is a nice abstraction! I think `queryDomain` might be a better name since it supports any query on the domains table that returns rows.